### PR TITLE
[890] Function "load_inline_svg" removed

### PR DESCRIPTION
**Pull request by rlaco-qualityunit**
**Changes proposed in this Pull Request**
Social icons in footer replaced with icons.svg.
Arrow in Affiliate Manager Cards block was replaced with icons.svg.
Function "load_inline_svg" removed.

**Testing instructions**
Check footer social icons.
Check arrow icon in Affiliate Manager Cards (e.g. [Affiliate Program Directory](https://www.postaffiliatepro.com/affiliate-program-directory/mobilecactus-affiliate-program/)).

**Checklist**
- [x] My code follows the style guidelines of this project
- [x] My commits follow the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

Close QualityUnit/web-issues#890

### CODE REVIEW

File: assets/styles/layouts/_Footer.scss [hunk 1]
```diff
@@ -138,19 +138,30 @@
...
+				&:active,
+				&:focus,
+				&:hover {
 
-				@include color(color, primary-color);
+					@include color(color, primary-color);
+					text-decoration: none;
 
-				svg path {
+					svg path {
```
Comment (jremen) [inline assets/styles/layouts/_Footer.scss:161]: svg without path is enough. If there is no fill color set (fill="none" is also considered as set) then svg takes currentColor specified via color: property

**Review by jremen — COMMENTED** [review]
svg without path is enough. If there is no fill color set (fill="none" is also considered as set) then svg takes currentColor specified via color: property

File: assets/styles/layouts/_Footer.scss (hunk 1 above)
Comment (rlaco-qualityunit) [reply assets/styles/layouts/_Footer.scss]: paht selector was removed

File: assets/styles/layouts/_Footer.scss [hunk 2]
```diff
@@ -138,19 +138,26 @@
...
+				&:active,
+				&:focus,
+				&:hover {
 
-				@include color(color, primary-color);
+					@include color(color, primary-color);
+					text-decoration: none;
 
-				svg path {
+					svg {
```
Comment (jremen) [inline assets/styles/layouts/_Footer.scss:157]: same, not needed to specify fill color

**Review by jremen — CHANGES_REQUESTED** [review]
fill not needed as SVG takes it as default fill: currentColor from parent element (in this case, from <a> tag

File: assets/styles/layouts/_Footer.scss (hunk 2 above)
Comment (rlaco-qualityunit) [reply assets/styles/layouts/_Footer.scss]: removed

**Review by jremen — APPROVED** [review]

**Pull request merged**